project_task_digitized_signature/report/project_task_report.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+<odoo>
+
+    <template id="report_taskorder_document" inherit_id="project_task_report.task_report" priority="9999">
```

Comment:
Don't put priority. Use it only when making `replace` operations to make sure they are performed after the rest of inheritance.